docs/MO_DG/prepare_model/convert_model/Convert_Model_From_TensorFlow.md

```diff
@@ -139,6 +139,16 @@ where the batch size and the sequence length equal 2 and 30 respectively.
 mo --saved_model_dir BERT --input mask,word_ids,type_ids --input_shape [2,30],[2,30],[2,30]
 ```
 
+## MO Python API Interface
```

Comment:
Conversion of TensorFlow models from the memory using Python API

Reply:
Done.